phoenix-core/src/main/java/org/apache/phoenix/coprocessor/UngroupedAggregateRegionObserver.java

```diff
@@ -1110,9 +1111,20 @@ private static PTable deserializeTable(byte[] b) {
 
     private RegionScanner rebuildIndices(final RegionScanner innerScanner, final Region region, final Scan scan,
                                          final RegionCoprocessorEnvironment env) throws IOException {
-
-        RegionScanner scanner = new IndexRebuildRegionScanner(innerScanner, region, scan, env, this);
-        return scanner;
+        if (!scan.isRaw()) {
```

Comment:
nit: assuming this is for rebuilding the history from data, a comment stating so would be useful.